xpad.c

```diff
@@ -1379,7 +1388,9 @@ static void xpad_set_up_abs(struct input_dev *input_dev, signed short abs)
 		break;
 	case ABS_Z:
 	case ABS_RZ:	/* the triggers (if mapped to axes) */
-		if (xpad->xtype == XTYPE_XBOXONE)
+		if (xpad->mapping & MAP_WHEEL)
```

Comment:
Can you please put a comment with details about which physical devices are mapped to which `ABS_` values and ranges and center of those values?